.travis.yml

```diff
@@ -15,28 +15,6 @@ before_install:
 
 install:
   - sudo apt-get install bazel
-  # TODO(fejta): vendor these things as go modules
-  - go get -u github.com/alecthomas/gometalinter
-  - go get -u github.com/bazelbuild/buildtools/buildifier
-  - gometalinter --install
 
 script:
-  # Build first since we need the generated protobuf for the govet checks
-  - bazel build --config=ci //...
   - ./hack/verify-all.sh
-  - buildifier -mode=check $(find . -name BUILD -o -name '*.bzl' -type f -not -wholename '*/vendor/*')
```

Comment:
it looks like buildifier and gometalinter checks were just removed?

Reply:
hack/verify-bazel.sh covers buildifier
nogo does many similar checks to gometalinter (which is getting turned down in a couple weeks)

Reply:
I don't think verify-bazel looks at bzl files though, does it?